themes/10up-theme/plop-templates/ui-component-css-index.hbs

```diff
@@ -0,0 +1 @@
+@import url("{{ kebabCase name }}");
```

Comment:
👍  I like that this starts with a compnent/index.css and component/component.css file. it feels tedious to start component css like this, but then you want to split your component/index.css into sub components and have to refactor.

Reply:
Thanks @ian-pvd, I went through exactly the same thought process and in the end it's proved super useful over time to work with this pattern.

Reply:
also @ian-pvd can I take a look at your custom component templates? I'd be interested to see how you approach them